test-packages/test-services-e2e/package.json

```diff
@@ -23,5 +24,10 @@
   "dependencies": {
     "bignumber.js": "^9.0.1",
     "moment": "^2.29.0"
+  },
+  "peerDependencies": {
+    "@sap-cloud-sdk/generator": "^2.3.0",
+    "@sap-cloud-sdk/util": "^2.3.0",
+    "@sap-cloud-sdk/odata-v4": "^2.3.0"
```

Comment:
[q] Is this intended?

Reply:
It is added because of the `depcheck`. I think it also makes sense to add it as "peerDependencies" since it will require the package that is using `test-services-e2e` to have these three dependencies, in our case `e2e-test` needs to have these dependencies in order to use the test services.